File: problem-13/prb13.c
#include<stdio.h>

int main()
{
    int n,i,a=0,b=0;
    scanf("%d", &n);
    if(n<1)
    {
        printf("\nINVALID INPUT");
    }
    else
    {
        for(i=1;i<=n;i++)
        {
            if(i%2!=0)
            {
                a=a+7;
            }
            else
            {
                b=b+6;
            }
        }
        if(n%2!=0)
        {
            printf("\n%d", a-7);
        }
        else
        {
            printf("\n%d", b-6);
        }
    }
    return 0;
}

File: problem-13/prb13.py
n = int(input())
a = 0 
b = 0

if(n < 1):
    print("INVALID INPUT")
else:
    for i in range(1, n):
        if(i % 2 != 0):
            a = a + 7
        else:
            b = b + 6
    if(n % 2 != 0):
        print(a)
    else:
        print(b)
